src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/netcoreapp/Version0/Collections.generated.txt

```diff
@@ -36,12 +36,12 @@ namespace Microsoft.Extensions.Configuration.Binder.SourceGeneration
     {
         #region IConfiguration extensions.
         /// <summary>Attempts to bind the configuration instance to a new instance of type T.</summary>
-        [InterceptsLocation(@"src-0.cs", 12, 17)]
+        [InterceptsLocation(@"src-0.cs", 13, 17)]
```

Comment:
Good to see this change in the "version 0" baseline - did you encounter any test issues or did you just remember to do this?